### PR TITLE
fix(esl-footnotes): esl-note is not respond on "ignore" attribute change

### LINKED ISSUE #1422: [🐛 in esl-note]: esl-note is not respond on "ignore" attribute change and do not have an api to request refresh
**Describe the 🐛 bug in component**
Can't control footnotes dynamically due to ignoring attributes update

**To Reproduce**
1. Setup environment
```js
document.querySelector('esl-note').setAttribute('ignore', 'all');
```
2. Check that the first footnote on the page disappeared immediately

**Current behavior**
Changing the attribute does not change the footnotes list

**Expected behavior**
Ignored footnote disappears immediately

### PR DESCRIPTION
**Pull request by dshovchko**
Closes #1422

### CODE REVIEW

**Review by fshovchko — APPROVED** [review]

File: src/modules/esl-footnotes/core/esl-note.ts [hunk 1]
```diff
@@ -114,11 +114,17 @@ export class ESLNote extends ESLBaseElement {
...
     }
     if (attrName === 'ignore') {
-      memoize.clear(this, 'queryToIgnore');
-      this.$$on(this._onBPChange);
+      this.updateIgnore();
     }
   }
 
+  /** Revise the settings for ignoring the note */
+  public updateIgnore(): void {
```
Comment (ala-n) [inline src/modules/esl-footnotes/core/esl-note.ts:122]: The only thing I'm concerned about is the name of that function, especially in the context of its public.
TBD: @exadel-inc/esl-core-team to discuss naming

**Review by ala-n — APPROVED** [review]

File: src/modules/esl-footnotes/core/esl-note.ts (hunk 1 above)
Comment (dshovchko) [reply src/modules/esl-footnotes/core/esl-note.ts:122]: I also hesitated with that name.
There was a great desire to call this method something like this `reviseIgnore()`

**Review by NastaLeo — APPROVED** [review]

File: src/modules/esl-footnotes/core/esl-note.ts (hunk 1 above)
Comment (ala-n) [reply src/modules/esl-footnotes/core/esl-note.ts:122]:
```suggestion
  public updateIgnoredQuery(): void {
```

**Review by yadamskaya — APPROVED** [review]

**Review by Natalie-Smirnova — APPROVED** [review]

**Review by ala-n — APPROVED** [review]

**Pull request merged**